pkg/commands/logging/bigquery/create.go

```diff
@@ -84,31 +84,31 @@ func (c *CreateCommand) ConstructInput(serviceID string, serviceVersion int) (*f
 
 	input.ServiceID = serviceID
 	input.ServiceVersion = serviceVersion
-	input.Name = c.EndpointName
-	input.ProjectID = c.ProjectID
-	input.Dataset = c.Dataset
-	input.User = c.User
-	input.Table = c.Table
-	input.SecretKey = c.SecretKey
+	input.Name = &c.EndpointName
```

Comment:
Hmm, we're being inconsistent here but maybe there is a reason that I'm not aware of...

e.g. `&someVariable` vs `fastly.String(someVariable)`.

Reply:
No- this is another tricky one- some packages used the fastly.X() a lot, and others didn't. I think the right way to do it would be to use fastly.X only when needing to create the pointer on the fly, i.e. not putting a variable into the fastly.X. Or in other words: `fastly.String("hello")` is ok, but `&myStringVar` should be used elsewhere? Does that make sense?

Reply:
Ah, I see. Yup, that makes sense. Thanks for clarifying 👍🏻